components/Dialogs/src/MDCAlertController.m

```diff
@@ -117,18 +117,42 @@ + (instancetype)alertControllerWithTitle:(nullable NSString *)alertTitle
   return alertController;
 }
 
++ (instancetype)alertControllerWithTitle:(nullable NSString *)alertTitle
+                       attributedMessage:(nullable NSString *)attributedMessage {
+  MDCAlertController *alertController = [[MDCAlertController alloc] initWithTitle:alertTitle
+                                                                attributedMessage:attributedMessage];
+
+  return alertController;
+}
+
```

Comment:
It looks like you’re not yet exposing this api. Please either do so or delete this addition for now.

Reply:
Holy smokes! That flew past my head. I'll expose it right now